packages/ckeditor5-engine/src/view/domconverter.js

```diff
@@ -850,7 +850,9 @@ export default class DomConverter {
 	 * @returns {Boolean}
 	 */
 	isDomSelectionBackward( selection ) {
-		if ( selection.isCollapsed ) {
+		// Due to a bug with .isCollapsed always returning true when the selection is from a shadowRoot, also check that the selection
```

Comment:
long line? what's the projects line length? are they using an auto-formatter?

Reply:
Their recommended line length is 120, but they have their maximum at 140. es-lint throws an error when it's over 140.

Reply:
120 and 140? 😲